app/views/shared/summary_cards/_group.html.slim

```diff
@@ -21,13 +21,18 @@
           = humanize_date(resource.published_at)
       .summary-card__metadata
         p
+          span.glyphicon.glyphicon-comment.glyphicon--right
+          span.disqus-comment-count data-disqus-identifier="#{url_for(resource)}" 
+            | 0 Comments
```

Comment:
OK, for the time being, don't worry about this. This isn't a dealbreaker and we shouldn't have it block the PR.